pkg/scaler/scaler.go

```diff
@@ -93,9 +96,13 @@ func NewController(clientset kubernetes.Interface,
 				// Two different versions of the same Deployment will always have different RVs.
 				return
 			}
+			klog.V(4).Info("Secret changed")
 			c.enqueueProbe(new)
 		},
-		DeleteFunc: c.enqueueProbe,
+		DeleteFunc: func(old interface{}) {
+			klog.V(4).Infof("Secret deleted\n")
+			c.enqueueProbe(old)
+		},
```

Comment:
How is this related to meltdown ? Just wanted to understand.

Reply:
This is done to understand what operation was triggered for probe to start. Currently there is no way to distinguish if the secret was updated, deleted or added and the context is lost.